controller/work_item_link_blackbox_test.go

```diff
@@ -103,8 +103,10 @@ func (s *workItemLinkSuite) cleanup() {
 	require.Nil(s.T(), db.Error)
 	db = db.Unscoped().Delete(&link.WorkItemLinkCategory{Name: "test-user"})
 	require.Nil(s.T(), db.Error)
-	db = db.Unscoped().Delete(&space.Space{ID: s.userSpaceID})
-	require.Nil(s.T(), db.Error)
+	if s.userSpaceID != uuid.Nil {
+		db = db.Unscoped().Delete(&space.Space{ID: s.userSpaceID})
+		require.Nil(s.T(), db.Error)
+	}
```

Comment:
I've put this in a separate PR as well 
https://github.com/almighty/almighty-core/pull/1105
So if we merge that, other PRs can consume that.

If #979 gets merged before #1105  
We may close #1105